spyder_kernels/customize/namespace_manager.py

```diff
@@ -78,6 +78,10 @@ def __enter__(self):
         if '__file__' in self.ns_globals:
             self._previous_filename = self.ns_globals['__file__']
         self.ns_globals['__file__'] = self.filename
+
+        # Add the current namedspace to the kernel so it can access
```

Comment:
Please change this to

    # Add the current namespace to the kernel so it can access it